src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -8,43 +8,30 @@
 
     <NoWarn>$(NoWarn);CA1001;CA1068;CA1069;CA1309;CA1310;CA1419;CA1711;CA1805;CA1806;CA1816;CA1822;CA1830;CA1834;CA1835;CA1836;CA1840;CA1844;CA2016;CA2101;CA2200;CA2201;CA2208;CA2219;CA2249;CA2251;CA5366;CS0168;CS0649;IDE0073;SYSLIB0004;nullable;</NoWarn>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-
-    <IsPartialFacadeAssembly Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">true</IsPartialFacadeAssembly>
-    <DefineConstants Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETStandard'">$(DefineConstants);netstandard;INTERNAL_NULLABLE_ATTRIBUTES</DefineConstants>
```

Comment:
I see that you removed the single usage of the FEATURE_TCPKEEPALIVE constant.
I assume you did not find any usage instances of INTERNAL_NULLABLE_ATTRIBUTES?

Reply:
`INTERNAL_NULLABLE_ATTRIBUTES` was only used by the .NETStandard implementation which doesn't exist anymore. `FEATURE_TCPKEEPALIVE` is only used by the .NETCoreApp build which is the only implementation, so I removed the ifdef around it.